remove reference to CONT in ADR1-3 per #94 (#96)
FamilySearch/GEDCOM: specification/gedcom.md

#### `ADR1` (Address Line 1) `g7:ADR1`

The first line of the address, used for indexing.

> This should have a payload with no line break and be equal to the the first line of the corresponding `ADDR`.

Author: ```suggestion
This structure should have a payload with no line break that is equal to the the first line of the corresponding `ADDR`.
```

> This should have a payload with no line break and be equal to the the first line of the corresponding `ADDR`.

Reviewer: ```suggestion
This should have a payload with no line break and be equal to the first line of the corresponding `ADDR`.
```

See `ADDRESS_STRUCTURE` for more.

#### `ADR2` (Address Line 2) `g7:ADR2`

The second line of the address, used for indexing.

> This should have a payload with no line break and be equal to the the second line of the corresponding `ADDR`.

Author: ```suggestion
This structure should have a payload with no line break that is equal to the the second line of the corresponding `ADDR`.
```

> This should have a payload with no line break and be equal to the the second line of the corresponding `ADDR`.

Reviewer: ```suggestion
This should have a payload with no line break and be equal to the second line of the corresponding `ADDR`.
```

See `ADDRESS_STRUCTURE` for more.

#### `ADR3` (Address Line 3) `g7:ADR3`

The third line of the address, used for indexing.

> This should have a payload with no line break and be equal to the the third line of the corresponding `ADDR`.

Author: ```suggestion
This structure should have a payload with no line break that is equal to the the third line of the corresponding `ADDR`.
```

> This should have a payload with no line break and be equal to the the third line of the corresponding `ADDR`.

Reviewer: ```suggestion
This should have a payload with no line break and be equal to the third line of the corresponding `ADDR`.
```
